site/docs/remote-execution.md

```diff
@@ -35,6 +35,8 @@ To run Bazel with remote execution, you can use one of the following:
     *   [Buildfarm](https://github.com/bazelbuild/bazel-buildfarm)
     *   [BuildGrid](https://gitlab.com/BuildGrid/buildgrid)
     *   [Scoot](https://github.com/twitter/scoot)
+*   Self-hosted or hosted
```

Comment:
Since this is commercial either way, I think this should just be under hosted, with a note that you can also self-host. Maybe the a better distinction is
Self-Service vs. Commercial  rather than manual vs. hosted.

Reply:
> Since this is commercial either way, I think this should just be under hosted, with a note that you can also self-host.

Done.

> Maybe the a better distinction is Self-Service vs. Commercial rather than manual vs. hosted.

SGTM, shall I change that in this PR?

Reply:
Sure.

Reply:
Done. I also removed "Commercial" from the Engflow description, since it is now under that section.